Formula/l/lesspipe.rb

```diff
@@ -24,7 +24,7 @@ def install
   def caveats
     <<~EOS
       add the following to your shell profile e.g. ~/.profile or ~/.zshrc:
-        export LESSOPEN="|#{HOMEBREW_PREFIX}/bin/lesspipe.sh %s"
+        export LESSOPEN="|$HOMEBREW_PREFIX/bin/lesspipe.sh %s"
```

Comment:
```suggestion
        export LESSOPEN="|#{HOMEBREW_PREFIX}/bin/lesspipe.sh %s"
```

Reply:
This suggestion completely erases the change.  Can you please explain why you don't agree with the pull request?

Reply:
Your version might not be set for all users.

We could also try `#{opt_bin}/lesspipe.sh`

Reply:
We do not export a `$HOMEBREW_PREFIX` variable in the users' environment.

Our caveats are typically written for the machine where they are displayed. Users who have specific use-cases can adapt for their specific needs.

Reply:
Ah, I see.  I thought the homebrew installer automatically updated the user's profile to include the homebrew environment variables, but I may be misremembering:

```
# Set PATH, MANPATH, etc., for Homebrew.
eval "$(/opt/homebrew/bin/brew shellenv)"
```